pkg/typescaffold/resource.go

```diff
@@ -42,7 +43,7 @@ func (r *Resource) Validate() error {
 	}
 
 	if len(r.Resource) == 0 {
-		r.Resource = flect.Pluralize(strings.ToLower(r.Kind))
+		r.Resource = inflection.Plural(strings.ToLower(r.Kind))
```

Comment:
It can broke the projects built so far and kubebuilder,
see: https://github.com/kubernetes-sigs/kubebuilder/blob/master/pkg/model/resource/utils.go#L59-L62

Reply:
/hold

Reply:
Agree. We can't make a change that leads to any different names here